aws/resource_aws_rds_cluster.go

```diff
@@ -232,7 +232,7 @@ func resourceAwsRDSCluster() *schema.Resource {
 							Type:         schema.TypeString,
 							Required:     true,
 							ForceNew:     true,
-							ValidateFunc: validateRdsIdentifier,
+							ValidateFunc: validateRestoreToPointInTimeSourceClusterIdentifier,
```

Comment:
The validations in the new func look like what we need! I would just suggest using `validation.Any()` here to incorporate the 2 conditions (like an `OR`) e.g.
```suggestion
							ValidateFunc: validation.Any(
							    validateArn,
							    validateRdsIdentifier),
```

Reply:
Thank you, I didn't know we already have such a useful function 👍